### PR TITLE
Se 420 clean up rule list

### PR DESCRIPTION
**Pull request by oscarsernarosero**
I would suggest looking at the [file in GitHub](https://github.com/thrackle-io/tron/blob/SE-420-Clean-up-rule-list/docs/userGuides/rules/RULE-GUIDE.md) instead of the raw markdown.

### CODE REVIEW

**Review by anne-thrackle — CHANGES_REQUESTED** [review]
Much better organization. 
Only thing I think we should change is to match the names/titles of the rules between the Rule Guide and the individual rule pages.

File: docs/userGuides/rules/RULE-GUIDE.md [hunk 1]
```diff
@@ -3,33 +3,33 @@
...
+| Token Volatility | TODO |
+| Transaction Restriction By Access Level | TODO |
+| Transaction Size Limits by Risk | TODO |
+| [Transfer Counter](./TRANSFER-COUNTER.md) | The transfer-counter rule enforces a daily limit on the number of trades for each token within a collection. In the context of this rule, a "trade" is a transfer of a token from one address to another. Example uses of this rule: to mitigate price manipulation of tokens in the collection via the limitation of wash trading or the prevention of malfeasance for holders who transfer a token between addresses repeatedly. When this rule is active and the tradesAllowedPerDay is 0 this rule will act as a pseudo "soulBound" token, preventing all transfers of tokens in the collection.   |
+| Transaction Size Per Time Period by Risk Score | TODO |
+| Withdrawal rule | TODO |
```
Comment (ShaneDuncan602) [inline docs/userGuides/rules/RULE-GUIDE.md]: I think we need to make sure the title of the rule in the guide matches the title in the rule detail page. For instance:
Account Balance by Risk Score != Account Balance By Risk Rule
Admin Withdrawal Rule != Application Admin Withdrawal
Maximum Account Balance By Access Level Rule != Balance by Access Level
Total Supply Volatility Rule != Circulating Supply Volatility
Minimum Account Balance By Date != Minimum Account Balance By Date Rule
Pause Window != Pause Rule
Token Trading Volume != Token Transfer Volume Rule
Transfer Counter != Transfer Counter Rule

**Review by ShaneDuncan602 — CHANGES_REQUESTED** [review]
Just some minor rule title matching.

File: docs/userGuides/rules/RULE-GUIDE.md (hunk 1 above)
Comment (anne-thrackle) [reply docs/userGuides/rules/RULE-GUIDE.md]: jinx

**Review by ShaneDuncan602 — CHANGES_REQUESTED** [review]
These two still don't quite match:
Account Balance by Risk Score != Account Balance By Risk
Max Balance by Access Level Rule != Maximum Account Balance By Access Level

**Review by ShaneDuncan602 — APPROVED** [review]
Looks great.

**Review by anne-thrackle — APPROVED** [review]

**Pull request merged**